README.md

```diff
@@ -1,93 +1,123 @@
-# AWS Serverless Application Model (AWS SAM)
+# AWS SAM transform
 
 [![Tests](https://github.com/aws/serverless-application-model/actions/workflows/build.yml/badge.svg)](https://github.com/aws/serverless-application-model/actions/workflows/build.yml)
 [![PyPI](https://img.shields.io/pypi/v/aws-sam-translator?label=PyPI)](https://pypi.org/project/aws-sam-translator/)
-[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/aws-sam-translator)](https://pypi.org/project/aws-sam-translator/)
-
-The [AWS Serverless Application Model](https://aws.amazon.com/serverless/sam/) (SAM) is an open-source framework for building serverless applications.
-
-It provides shorthand syntax to express functions, APIs, databases, and event source mappings.
-With just a few lines of configuration, you can define the application you want and model it.
-
-## Recent blogposts and workshops
-
```

Comment:
nit: should we move this to the end? Not going to block on this, as we can iterate.

Reply:
I added the SAM Accelerate blog post. The other ones are quite SAM CLI-only specific, but will revisit these later.